tdrs-frontend/src/components/Header/Header.test.js

```diff
@@ -48,52 +48,6 @@ describe('Header', () => {
     expect(menuBtn).toExist()
   })
 
-  it('should call toggleMobileNav when menu button is clicked on mobile', () => {
```

Comment:
while the test setup would be different it seems like these would make good assertions to still have.

Reply:
@lauraGgit I tried for hours and hours both last week and today to get these tests to work. I have never gotten them passing so if you have any suggestions that would be great. The only thing I really have access to is the added class of `is-visible` and that class never shows up in the test.